app/Modules/Bookings/BookingsService.php

```diff
@@ -8,6 +8,8 @@
 use App\User;
 use Carbon\Carbon;
 
+/** TODO: Users can view/amend other users bookings. We need to make sure we fix this. */
```

Comment:
Not quite sure what you mean by this TODO. User's are limited to editing their own bookings, as the authenticated user is used in the API methods as the user argument.

Reply:
I'll investigate, I'm sure when I tested it I could amend other users bookings..